org.eclipse.ice.archetypes/data-element-archetype/src/main/resources/archetype-resources/src/main/java/App.java

```diff
@@ -0,0 +1,13 @@
+package $package;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
+    public static void main( String[] args )
+    {
+        System.out.println( "Hello World!" );
```

Comment:
It'd be cool to have an example DataElement specification class included in the archetype. Perhaps something like:

```java
package $package

import org.eclipse.ice.dev.annotations.DataElement;
import org.eclipse.ice.dev.annotations.DataField;

@DataElement(name = "Example")
public class ExampleSpec {
	@DataField public String exampleField;
}
```

Reply:
Oh, maybe even something showing some interaction with DataElements in the generic `main` method would be kind of nice. Thoughts @jayjaybillings?

Reply:
I agree, and furthermore I think it should include a DataElement and a DataModel. We can think about whether or not there should be a bare bones generator and one that just adds the examples, but we should definitely have them.

Reply:
Ok. I'll create some general classes along these lines.